### PR TITLE
Updated education levels to new API format.

### PR DESCRIPTION
**Pull request by dsjen**
- added translation to education levels
- updated education chart and table text

edu

@rocha @clintonb @lamagnifica @shnayder @marcotuts

### CODE REVIEW

File: analytics_dashboard/courses/presenters.py [hunk 1]
```diff
@@ -549,13 +564,25 @@ def _build_education_levels(self, api_response):
...
-                'educationLevelShort': 'Unknown',
-                'educationLevelLong': 'Unknown',
+                'educationLevel': UNKNOWN_EDUCATION_LEVEL_NAME,
                 'count': unknown_count
             })
 
         return levels
 
+    def _fill_empty_education_levels(self, api_response):
+        found_levels = [level['education_level'] for level in api_response]
```
Comment (clintonb) [inline analytics_dashboard/courses/presenters.py:624]: This is where grouping the API data by course and date help downstream.

File: analytics_dashboard/courses/views.py [hunk 2]
```diff
@@ -405,7 +399,7 @@ def get_context_data(self, **kwargs):
         context['js_data']['course']['enrollmentTrends'] = trend
 
         context.update({
-            'page_data': json.dumps(context['js_data']),
+            'page_data': json.dumps(context['js_data'], cls=LazyEncoder),
```
Comment (clintonb) [inline analytics_dashboard/courses/views.py]: This smells. At a minimum refactor this bit into a method, `get_page_data(self, context)`, and set `context['page_data']` after this call to `update`.

Comment (dsjen) [inline analytics_dashboard/courses/views.py]: Added `LazyEncoderMixin` which has `get_page_data`.

File: analytics_dashboard/courses/presenters.py (hunk 1 above)
Comment (dsjen) [inline analytics_dashboard/courses/presenters.py:624]: Yes, this could be cleaned up with with that change. :(

**Comment by clintonb** [comment]
:+1:

**Comment by rocha** [comment]
:rocket:

**Pull request merged**